github/ci/prow-deploy/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits-0.99.yaml

```diff
@@ -0,0 +1,329 @@
+---
+presubmits:
+  kubevirt/cluster-network-addons-operator:
+    - name: pull-e2e-cnao-lifecycle-k8s-release-0.99
+      branches:
+        - release-0.99
+      always_run: true
+      decorate: true
+      decoration_config:
+        timeout: 4h
+        grace_period: 5m
+      max_concurrency: 6
+      cluster: prow-workloads
+      labels:
+        preset-podman-in-container-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-shared-images: "true"
+      spec:
+        nodeSelector:
+          type: bare-metal-external
+        containers:
+          - image: quay.io/kubevirtci/bootstrap:v20230801-94954c0
+            securityContext:
+              privileged: true
+            resources:
+              requests:
+                memory: "52Gi"
```

Comment:
wow, this is a lot
maybe we need to reduce it all over?

same in the ones below

Reply:
I think that the tier1 tests that don't deploy much - it's totally doable
In the workflow and lifecycle - I would stick to 52Gi, just because it's what HCO [does](https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/files/jobs/kubevirt/hyperconverged-cluster-operator/hyperconverged-cluster-operator-presubmits-2.4.yaml#L30).

In order to avoid too much noise, do you mind if we do it on a separate PR, and only to the main branch jobs? 
Once we see it's stable, we can move to the other jobs..

Reply:
problem is that this doesnt allow other jobs to run maybe, we can double check with Brian that i am not wrong
we should stick to what we did until now on CNAO imho, not HCO
if this is already the state on previous lanes, sure we can do it on follow-up / start with just main please

Reply:
yeah, I didn't increase the memory for this PR. This is why I suggest to reduce (first in main, then on the others) in a follow up PR